FirebaseCore/Sources/FIRApp.m

```diff
@@ -836,11 +836,20 @@ + (void)registerSwiftComponents {
   SEL componentsToRegisterSEL = @selector(componentsToRegister);
   // Dictionary of class names that conform to `FIRLibrary` and their user agents. These should only
   // be SDKs that are written in Swift but still visible to ObjC.
-  NSDictionary<NSString *, NSString *> *swiftLibs = @{@"FIRFunctionsComponent" : @"fire-fun"};
-  for (NSString *className in swiftLibs.allKeys) {
+  NSDictionary<NSString *, NSString *> *swiftComponents = @{@"FIRFunctionsComponent" : @"fire-fun"};
+  for (NSString *className in swiftComponents.allKeys) {
     Class klass = NSClassFromString(className);
     if (klass && [klass respondsToSelector:componentsToRegisterSEL]) {
-      [FIRApp registerInternalLibrary:klass withName:swiftLibs[className]];
+      [FIRApp registerInternalLibrary:klass withName:swiftComponents[className]];
+    }
+  }
+
+  // Swift libraries that don't need component behaviour
+  NSDictionary<NSString *, NSString *> *swiftLibraries = @{@"FIRCombineSDK" : @"fire-comb"};
```

Comment:
How about "fire-co-auth", etc

Reply:
Agreed except I think we had the goal to keep these strings less than 9 characters. @ncooke3 Would you check with the metrics team?

Reply:
We discussed a bit and realized that the "fire-" doesn't provide any value, especially because "fire-auth" is already there. How about "comb-auth", etc?

Reply:
SG!

(Reminds me of the 8.3 filename requirements the DOS filesystem imposed ;-) )